gapic/templates/%namespace/%name_%version/%sub/services/%service/client.py.j2

```diff
@@ -30,8 +30,13 @@ from google.iam.v1 import policy_pb2 as policy  # type: ignore
 {% endif %}
 {% endfilter %}
 from .transports.base import {{ service.name }}Transport, DEFAULT_CLIENT_INFO
-from .transports.grpc import {{ service.grpc_transport_name }}
-from .transports.grpc_asyncio import {{ service.grpc_asyncio_transport_name }}
+{%- if 'grpc' in opts.transport %}
+from .transports.grpc import {{ service.name }}GrpcTransport
```

Comment:
This seems like a non-trivial stylistic change, from having the transport name in a variable to constructing it in the template. I'm OK if @software-dov is fine with it, but just calling it out to make sure this is intentional.

Reply:
I'm more in favor of keeping it the way it was, with 'GrpcTransport' provided by the service's `grpc_transport_name` property.